src/app/stores/scores.store.ts

```diff
@@ -0,0 +1,5 @@
+import { Store } from '../../common/store';
+
+export class ScoresStore extends Store {
```

Comment:
what is this for?

Reply:
Oh shoot, left over code. Ended up not needing it